React/wbooks/src/Navbar/layout.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+import logo from '../assets/wbooks_logo.svg';
+import notificationsImg from '../assets/notifications.svg';
+import addBookImg from '../assets/add_book.svg';
+import profileImg from '../assets/cat_no_banana.jpeg';
+import './style.css';
+
+import Menu from './Components/Menu';
+import Notifications from './Components/Notifications';
+
+function NavBar({ logoutHandler, isMenuDisplayed, areNotificationsDisplayed, onToogleNotif, onToogleMenu }) {
+  const menu = isMenuDisplayed ? <Menu logoutHandler={logoutHandler} /> : undefined;
```

Comment:
we usually return `null` instead of undefined when we don't want to render anything. It is defined, it's just empty.